### ISSUE TITLE
[FEATURE_REQUEST] Amoled Black theme needed

### ISSUE THREAD

**Issue by Black-StellarSpectrum** [mateusz-bak/openreads-android#271]
**Is your feature request related to a problem? Please describe.**
Not a problem per say. 

**Describe the solution you'd like**
I'd like to have Amoled black theme.  And instead of preset colours for accent why not just let us add hex code of the colour we want! 

**Describe alternatives you've considered**
None. 

**Additional context**

**Comment by exstntlmsnthrp** [comment]
I agree with letting the user input hex code for color choice as an additional option!

**Comment by mateusz-bak** [comment]
Implementing in #280  and #285 
Will be implemented in the next release.
